pkg/gadgets/audit-seccomp/tracer/bpf/audit-seccomp.c

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: GPL-2.0
+/* Copyright (c) 2021 The Inspektor Gadget authors */
+
+/* This BPF program uses the GPL-restricted function bpf_probe_read*().
+ */
+
+#include <vmlinux/vmlinux.h>
+
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+
+#include "audit-seccomp.h"
+
+#include <gadgettracermanager/bpf-maps.h>
+
+/* The stack is limited, so use a map to build the event */
+struct {
+	__uint(type, BPF_MAP_TYPE_PERCPU_ARRAY);
+	__uint(max_entries, 1);
+	__type(key, __u32);
+	__type(value, struct event);
+} tmp_event SEC(".maps");
+
+struct {
+	__uint(type, BPF_MAP_TYPE_PERF_EVENT_ARRAY);
+} events SEC(".maps");
+
+SEC("kprobe/audit_seccomp")
+int kprobe__audit_seccomp(struct pt_regs *ctx)
+{
+	unsigned long syscall = PT_REGS_PARM1(ctx);
+	int code = PT_REGS_PARM3(ctx);
+
+	struct task_struct *task = (struct task_struct *)bpf_get_current_task();
+	__u64 mntns_id = BPF_CORE_READ(task, nsproxy, mnt_ns, ns.inum);
+	if (mntns_id == 0) {
```

Comment:
Did you give this eBPF CO-RE to iovisor/bcc?
If not, can you please remove the brackets here to try to follow kernel coding style.